package.json

```diff
@@ -1,50 +1,89 @@
 {
-    "name": "meson",
```

Comment:
I might be mistaken, but package.json should always be indented with 2 spaces. Otherwise `npm install --save DEP` will rewrite the entire file changing whatever indentation to 2 spaces.

The current file is using 4 spaces, maybe we should fix this once and for all?